src/celengine/render.h

```diff
@@ -8,19 +8,20 @@
 // as published by the Free Software Foundation; either version 2
 // of the License, or (at your option) any later version.
 
-#ifndef _CELENGINE_RENDER_H_
-#define _CELENGINE_RENDER_H_
+#pragma once
 
 #include <list>
 #include <memory>
 #include <string>
 #include <vector>
+
 #include <Eigen/Core>
-#include <celengine/universe.h>
-#include <celengine/selection.h>
-#include <celengine/starcolors.h>
-#include <celengine/rendcontext.h>
-#include <celengine/renderlistentry.h>
+
+#include "universe.h"
+#include "selection.h"
+#include "starcolors.h"
+#include "rendcontext.h"
+#include "renderlistentry.h"
```

Comment:
i think it's better to restore the original includes because i have an idea to separate render code from other engine parts, it will include celengine/foo.h and the change  will be smaller